src/lzma_worker.js

```diff
@@ -2628,7 +2628,8 @@ var LZMA = (function () {
     return {
         /** xs */
         compress:   compress,
-        decompress: decompress
+        decompress: decompress,
+        terminateWorker: function() {}
```

Comment:
Exactly, I thought it might be good to have a consistent API across the platforms. I agree that it probably does not make a lot of difference, since this version of `terminateWorker` is only used when using Node.js or similar, where the code author will know for sure that there are no web workers involved.

I can’t really tell how much code using this module is expected to run both server-side and client-side; If you don’t think it’s a notable amount, you can just drop it.